test/edu-benefits/1990e/schema/maximal-test.json

```diff
@@ -34,9 +34,11 @@
         "last": "Lkjh",
         "suffix": "Jr."
       },
-      "veteranSocialSecurityNumber": "556789999",
-      "view:noSSN": true,
-      "vaFileNumber": "C12345678",
+      "view:veteranId": {
```

Comment:
I think I wrote this before adding the `veteranId` stuff.  I need to go back and understand how the schema validation works to see why no tests complained.